spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index', type: :requests do
```

Comment:
- [ ] For the Post index page test, you seem to have skipped the following test scenarios. Kindly add them:
  
  - Tests that asserts you can see some of the post's body.
  - Tests that redirects to the post's show page when you click on a post.

Reply:
done